lib/core/commands/graphic_factory/graphic_factory.dart

```diff
@@ -16,6 +16,20 @@ class GraphicFactory {
     ..color = const Color.fromARGB(220, 117, 117, 117)
     ..style = PaintingStyle.fill;
 
+  static Paint boundingBoxPaint = Paint()
```

Comment:
Please merge develop to get the changes from shapes tool PR. There these are already written/implemented